.gitignore

```diff
@@ -92,3 +92,7 @@ public/sitemap*.xml.gz
 # secrets we need for dev and test, so
 # we really don't want this in version control
 .env.*.local
+env.*.local
```

Comment:
These changes aren't really related to the login button, can we remove them for now? I'm super happy to bring them back in a separate PR, but I'd like to keep each PR _atomic_ (i.e., it only changes/does one thing)